Repo.cs

```diff
@@ -9,14 +9,18 @@ static class Repo
         private static Lazy<Dictionary<string, string[]>> _meatPizzas = new Lazy<Dictionary<string, string[]>>(() =>
             new Dictionary<string, string[]>()
             {
-                { "Meat Lovers",        new[] { "pepperoni", "beef", "sausage", "ham" } },
-                { "Hawaiian",           new[] { "ham", "pineapple" } },
-                { "Deluxe Hawaiian",    new[] { "ham", "bacon", "pineapple" } },
-                { "Hawaiian Elvis",     new{} { "canadian bacon", "pineapple", "saurkraut"}},
-                { "Supreme",            new[] { "pepperoni", "sausage", "mushroom", "green pepper", "onion" } },
-                { "Pepperoni Bacon",    new[] { "pepperoni", "bacon" } },
-                { "Beef Lovers",        new[] { "pepperoni", "beef" } },
-                { "Pepperoni Veggie",   new[] { "pepperoni", "onion", "green pepper" } }
+                { "Meat Lovers",              new[] { "pepperoni", "beef", "sausage", "ham" } },
+                { "Hawaiian",                 new[] { "ham", "pineapple" } },
+                { "Deluxe Hawaiian",          new[] { "ham", "bacon", "pineapple" } },
+                { "Hawaiian Elvis",           new[] { "canadian bacon", "pineapple", "sauerkraut"} },
+                { "Supreme",                  new[] { "pepperoni", "sausage", "mushroom", "green pepper", "onion" } },
+                { "Pepperoni Bacon",          new[] { "pepperoni", "bacon" } },
+                { "Beef Lovers",              new[] { "pepperoni", "beef" } },
+                { "Pepperoni Veggie",         new[] { "pepperoni", "onion", "green pepper" } },
+                { "Prosciutto and Arugula",   new[] { "mozzarella chesse", "parmesan", "prosciutto", "arugula" } },
+                { "For Seasons",              new[] {"tomato", "mozzarella cheese", "basil leaves", "artichoke",

```

Comment:
I'm guessing this is supposed to be Four Seasons?
```suggestion
                { "Four Seasons",              new[] {"tomato", "mozzarella cheese", "basil leaves", "artichoke",
```